config/routes.rb

```diff
@@ -30,6 +30,10 @@
         resources :courses
       end
     end
+
+    namespace :v2 do
+      resources :users
```

Comment:
Should we restrict the routes to match the controller actions? i.e. `resources :users, only: :show`

Reply:
I don't know. I agree that it would be more "correct", but I think the behaviour if we don't is fine (it'll still generate 404s, right?), and it looks a bit tidier and easier to read in the routes file.

The 2 really convincing arguments for restricting the routes are when you have a large number of routes where you might get a speed benefit, or when you have the actions available in the controller but you don't want to expose them for some reason.

Reply:
I say we aim for legibility for now.